### PR TITLE
Replaced unwrap with proper error handling on create_image

### PR DESCRIPTION
**Pull request by ElArtista**
Fixes crash when client asks for unsupported texture type

### CODE REVIEW

File: src/backend/gl/src/device.rs [hunk 1]
```diff
@@ -1402,7 +1402,11 @@ impl d::Device<B> for Device {
     ) -> Result<n::Image, i::CreationError> {
         let gl = &self.share.context;
 
-        let desc = conv::describe_format(format).unwrap();
+        let desc;
+        match conv::describe_format(format) {
```
Comment (kvark) [inline src/backend/gl/src/device.rs:1406]: let's do it with expressions:
```rust
let desc = conv::describe_format(format).ok_or(i::CreationError::Format(format))?;
```

Comment (ElArtista) [reply src/backend/gl/src/device.rs:1406]: Yes, much better. Learning better Rust in the way!

**Comment by kvark** [comment]
Please squash the commits as well: our merge bot doesn't know how to do it.

**Comment by ElArtista** [comment]
Its ready.

**Review by kvark — APPROVED** [review]
bors r+

**Pull request merged**